RevPiDevice.c

```diff
@@ -243,7 +244,16 @@ int RevPiDevice_run(void)
 
 	// if the user-ioctl want to send a telegram, do it now
 	if (test_bit(REVPI_CORE_FLAG_PENDING_TELEGRAM, &piCore_g.flags)) {
-		piCore_g.statusUserTel = piIoComm_sendTelegram(&piCore_g.requestUserTel, &piCore_g.responseUserTel);
+		SIOGeneric *telegram = &piCore_g.requestUserTel;
+		UIoProtocolHeader *hdr = &telegram->uHeader;
+		char rcv_buff[IOPROTOCOL_MAXDATA_LENGTH];
+
+		piCore_g.statusUserTel = pibridge_req_io(hdr->sHeaderTyp1.bitAddress,
```

Comment:
This seems to exceed 80 chars. Maybe move the `pibridge_req_io(hdr->sHeaderTyp1.bitAddress,` to the next line and indent with 3 tabs? Can't really think of a better solution.